roles/satellite-clone/files/disassociate_capsules.rb

```diff
@@ -11,42 +12,39 @@ def prepare_hammer_cmd(command)
 end
 
 def run_hammer_cmd(command)
-  command = prepare_hammer_cmd(command)
-  `#{command}`
+  `#{prepare_hammer_cmd(command)}`
 end
 
-def get_info_from_hammer(command, column=1)
-  bash_parse = " | grep -v \"Warning:\" | tail -n+2 | awk -F, {'print $#{column}'}"
```

Comment:
while the pipes and the greps and the tails are awkward, the new code will fail if hammer outputs e.g. this:

```
# hammer --output json proxy list
NOTE: Gem.gunzip is deprecated; use Gem::Util.gunzip instead. It will be removed on or after 2018-12-01.
Gem.gunzip called from /usr/lib/ruby/vendor_ruby/unicode/display_width/index.rb:5.
[
  {
    "Id": 1,
    "Name": "ubuntu2004-foreman-3-0.yatsu.example.com",
    "URL": "https://ubuntu2004-foreman-3-0.yatsu.example.com:8443",
    "Features": [
      "Puppet CA",
      "Puppet",
      "Logs",
      "Registration"
    ]
  }
]
```

The hammer call should at least gain a `2>/dev/null` to drop stderr.

Reply:
:+1: thanks for the reminder to do this